src/ResponsiveProvider.js

```diff
@@ -11,6 +10,7 @@ const ResponsiveProvider = ({
     breakpoints,
     breakpointsMax,
     mediaQueries,
+    mobileBreakpoint,
```

Comment:
Not sure about adding another prop.  Especially since the concept of a mobile breakpoint isn't completely separate from the meanings of the media types. 

For example, I think everyone can agree that nobody would ever choose, `lg` or `xl` as mobile.  Similarly usually `s` and `xs` are very likely to be mobile.  And `md` seems to have the semantics of "this is where the switch happens".  If we introduce the concept of "mobileBreakpoint" then we risk encouraging semantic inconsistencies such as `xl` being mobile.  

Technically, anyone can choose their mobile breakpoint just by choosing what they want their `md` to be.  If we want more customization, then I would prefer allowing more breakpoints, such as `md-lg`, and `sm-md` (not even sure if this is a thing).  But I'd still rather wait for a use case rather than err on the side of speculative generality.

Reply:
We have `breakpoints` `breakpointsMax` but in both of those we dont have the concept of where is `mobile` and where's not.

I think the clients of the library will probably never do `lg` or `xl` but we give them the oportunity to choose. 

Right now is fixed at `md` and that's not good for a customizable library.

Maybe after we could rethink how we pass the breakpoints and cover already the `mobileBreakpoint` to be fewer props, but we also need to make it easy to understand.

Reply:
I'm seeing this from the perspective of, every new feature is a breaking change in disguise.   Each addition to the API is another potential for a breaking change in the future.  If we don't have a strong reason to add the feature and it can be achieved through other means, we should probably hold off. 

I see `md` not being customizable as a good thing, it helps ensure consistency in the meaning of the media types.  I don't think a library should be infinitely flexible/customizable, it should be flexible in the right places.   Right now, you just choose the correct `pixels/rem` value for `md`.

I think some work needs to be done on the` breakpointsMax` and `mediaQueries` parameters, they seem redundant and unnecessary, but maybe I'm missing something...

Reply:
Let's see what the others think about this feature if we should close the MR / Issue or something else @SoaresMG  @sofiacteixeira  @themariamarques @themariamarques

Reply:
@stephan-noel, I get what you are saying but you have to think that the [xs, sm, md, lg, xl] implementation isn't always used, that's up to the developer using the package, we shouldn't hinder his architecture decisions just because we don't want one more prop.

We should try to stop changing the public API (properties, public names, etc) however this one is a fix since it should be like this since the initial changes we made.

Reply:
With this being changeable anyone can use the library even if he doesn't use the common [xs, sm, md, lg, xl] breakpoints.

Reply:
We could perhaps add default breakpoints and therefore we could have a default initialMediaType and a default mobileBreakpoint

Reply:
I think that would a good idea honestly.

Reply:
@SoaresMG thanks for the response.  Yes I see what you mean, I don't think we need to wait for every use case to come up before we take it into account.   I was hoping that some clarification on the `breakpoints`, `mediaQueries`, `maxBreakpoints` could be reached before adding this so that this prop could be considered in that larger context.  But if everyone is onboard, go for it 👍

Reply:
Yes, let's go forward with it and we may open an issue in the future to redo the defaults